authz/constants.go

```diff
@@ -46,3 +46,11 @@ var DefaultAuthZEdgeTypes = []EdgeType{
 		},
 	},
 }
+
```

Comment:
Lets manually delete these - we don't need them in the SDK right ?

Reply:
ok